groups/createElements.js

```diff
@@ -161,13 +173,17 @@ const createGroupCreationModal = (onClose = () => {}, onSubmit = () => {}) => {
 
   form.onsubmit = (e) => {
     e.preventDefault();
-    const inputValue = getInput();
-    if (inputValue.length === 0 || inputValue.includes('group')) {
+    const title = getTitle();
+    const description = descriptionField.value.trim();
+    if (title.length === 0 || title.includes('group')) {
       throw new Error('Invalid group name');
     }
 
     toggle({ enabled: false });
-    onSubmit(inputField.value).then(() => {
+    onSubmit({
+      title,
+      description,
+    }).then(() => {
```

Comment:
@pallabez Add validation for the description field. Check for input length, special characters, or prohibited words.